core/storage/user/gae_models.py

```diff
@@ -2616,7 +2617,9 @@ def get_model_association_to_user():
 
     @classmethod
     def get_export_policy(cls):
-        """DeletedUserModel contains only IDs that were deleted."""
+        """Model doesn't contain any data directly corresponding to a user.
```

Comment:
Suggested change: ``..to a particular, existing user.`` (to further emphasize that the deleted IDs no longer correspond to users).

Reply:
Done